public/locations-list-map.js

```diff
@@ -570,11 +572,15 @@ function centerMapToMarkersNearCoords(latitude, longitude) {
  * Changes the markers currently rendered on the map based strictly on . This will reset the 'markers' module variable as well.
  */
 function showMarkers(data, filters, showNearest) {
+
   markers = [];
 
   if (!map) {
     return;
   }
+  if (markerCluster){
```

Comment:
If you're worried about markerCluster not existing, add it to the 'if (!map)' check up above. You're using markerCluster blindly down below when adding the markers, so checking it like this doesn't buy you anything.

Reply:
@drwestco suggest a change?

Reply:
fixing this rn!